tests/xharness/Logging/CaptureLog.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.IO;
+
+namespace Xharness.Logging {
+
+	// A log that captures data written to a separate file between two moments in time
+	// (between StartCapture and StopCapture).
+	public class CaptureLog : Log
+	{
+		public string CapturePath { get; private set; }
+		public string Path { get; private set; }
+
+		long startPosition;
+		long endPosition;
+		bool entire_file;
```

Comment:
I think you should stick with camelCase rather than snake_case just like rest of the code base

Reply:
old code. But is a good time to unify.